v2/template.go

```diff
@@ -29,6 +30,22 @@ type Template struct {
 	Zone            *string
 }
 
+// Implement sort.Interface for []*Template by CreatedAt or by Nane
+type Templates []*Template
+
+func (s Templates) Len() int      { return len(s) }
+func (s Templates) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
```

Comment:
Just use generics, https://pkg.go.dev/golang.org/x/exp/slices#Sort

Reply:
This project is still buildable with go 1.17 for now; generics are available from 1.18.x

Reply:
1.20 is around the corner. It's likely you won't lose anything by bumping it to 1.18.

Reply:
We will do it later